### DISCUSSION TITLE
兩周年紀念日 vs 兩周年祭念日

### DISCUSSION THREAD

**Discussion by lyykey** [1101-GS4719-C/Python-Community-GS4719-C#1932] [General]
大家好，小弟我面臨極大難題，比資工系何老先生演算法還難。

再過一個禮拜就要在一起兩周年，希望各位仁兄能夠江湖救濟，來日有難 避 出手相救！

不知道大家對於情侶兩周年紀念日怎麼過有什麼想法，桃園有什麼好吃好玩的都可以ㄉ，但我真的該去的好像都去過了！

不要跟我講寫卡片！第一年怕分手，寫的很勤奮。現在已經到互相不寫卡片也能體會對方忘記了。重點是該說的土味情話都說完了。

謝謝大家的回覆，共勉之！

**Comment by SusuTima** [comment]
帶她聽何老先生的演算法，讓你們一起徜徉在演算法的世界裡。

**Comment by benbebobo149** [comment]
我覺得你的問題非常棒，但根據經驗法則在網路上問類似的問題，只會得到一個回復「感情問題一律建議分手」。我建議你可以嘗試找尋更具體和實際的建議，或者考慮尋求專業的諮詢。這樣可能會更有助於解決你遇到的難題。希望同學可以度過這個難關。感謝同學的分享，這讓我反思了在面對困難時該如何處理和尋求幫助的重要性。

**Comment by 20050201Shawn** [comment]
我應該會跟他宅在家裡不出門，一起耍廢一整天。雖然出去玩很開心但回家身體會很疲憊，不如就在家一起養老，嚕貓吸貓，叫uber吃頓好吃的，隨便點開一部youtube影片，一起躺在床上邊吃東西邊看影片是再浪漫不過。

**Comment by hbh10601** [comment]
如果願意騎車40分鐘的話，非常推薦大古山的151高地，那邊的夜景應該是全桃園最頂，沒去過的話蠻值得去一下的，大推。
可以從下午先去竹圍或永安漁港吃個生魚片、看看海，然後晚上剛好從那邊上山看夜景，送個簡單的花就讚讚。
